src/main/java/uk/gov/hmcts/reform/em/hrs/service/email/SummaryReportService.java

```diff
@@ -31,13 +33,19 @@ public SummaryReportService(
         HearingRecordingStorage hearingRecordingStorage
     ) {
         this.emailSender = emailSender;
-        this.recipients = recipients;
+        if (recipients == null || recipients.length == 0) {
+            throw new RuntimeException("No recipients configured for reports");
```

Comment:
I assume this should be discussed with the business

Reply:
to send email we need recipients and task omits exceptions

Reply:
is it possible not to send report if there are no recipients defined? or not sending reports means application should not run?

Reply:
report sending task is configurable if it is enabled we need to make this error visible .